MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -471,6 +471,58 @@ public virtual void Unload()
 		    loadedAssets.Clear();
 		}
 
+        /// <summary>
+        /// Unloads a single asset.
+        /// </summary>
+        /// <param name="assetName">The name of the asset to unload. This cannot be null.</param>
+        public virtual void UnloadAsset(string assetName)
```

Comment:
Does overloading `Unload` make more sense than `UnloadAsset`, since the load method is `Load`, not `LoadAsset`?